network.go

```diff
@@ -54,6 +54,10 @@ type Network interface {
 
 	// Return certain operational data belonging to this network
 	Info() NetworkInfo
+
+	// Peers returns the IP addresses of the peer nodes participating in the same overlay network
+	// For non-dynamic overlay networks and bridge networks it returns an empty slice
+	Peers() []string
```

Comment:
`Peers is not part of the original network abstraction.` - We add additional methods to interfaces when there is a need. Its irrelevant what the original abstraction was.  Overlay network has peers in the non-swarm mode as well; but we weren't displaying it.  This is a cosmetic issue. I will move it to the `NetworkInfo` interface.